Audio-Transcription-Firefox/content.js

```diff
@@ -0,0 +1,283 @@
+        elem_text.id = "t" + i;
+        elem_container.appendChild(elem_text);
+
+        if (i == 3) {
+            elem_text.style.top = "-1000px"
+        }
+    }
+
+    document.body.appendChild(elem_container);
+
+    let x = 0;
+    let y = 0;
+
+    // Query the element
+    const ele = elem_container;
+
+    // Handle the mousedown event
+    // that's triggered when user drags the element
+    const mouseDownHandler = function (e) {
+        // Get the current mouse position
+        x = e.clientX;
+        y = e.clientY;
+
+        // Attach the listeners to `document`
+        document.addEventListener('mousemove', mouseMoveHandler);
+        document.addEventListener('mouseup', mouseUpHandler);
+    };
+
+    const mouseMoveHandler = function (e) {
+        // How far the mouse has been moved
+        const dx = e.clientX - x;
+        const dy = e.clientY - y;
+
+        // Set the position of element
+        ele.style.top = `${ele.offsetTop + dy}px`;
+        ele.style.left = `${ele.offsetLeft + dx}px`;
+
+        // Reassign the position of mouse
+        x = e.clientX;
+        y = e.clientY;
+    };
+
+    const mouseUpHandler = function () {
+        // Remove the handlers of `mousemove` and `mouseup`
+        document.removeEventListener('mousemove', mouseMoveHandler);
+        document.removeEventListener('mouseup', mouseUpHandler);
+    };
+
+    ele.addEventListener('mousedown', mouseDownHandler);
+}
+
+function getStyle(el,styleProp)
+{
+    var x = document.getElementById(el);
+    if (x.currentStyle)
+        var y = x.currentStyle[styleProp];
+    else if (window.getComputedStyle)
+        var y = document.defaultView.getComputedStyle(x,null).getPropertyValue(styleProp);
+    return y;
+}
+
+function get_lines(elem, line_height) {
+    var divHeight = elem.offsetHeight;
+    var lines = divHeight / line_height;
+
+    var original_text = elem.innerHTML;
+
+    var words = original_text.split(' ');
+    var segments = [];
+    var current_lines = 1;
+    var segment = '';
+    var segment_len = 0;
+    for (var i = 0; i < words.length; i++)
+    {
+        segment += words[i] + ' ';
+        elem.innerHTML = segment;
+        divHeight = elem.offsetHeight;
+
+        if ((divHeight / line_height) > current_lines) {
+            var line_segment = segment.substring(segment_len, segment.length - 1 - words[i].length - 1);
+            segments.push(line_segment);
+            segment_len += line_segment.length + 1;
+            current_lines++;
+        }
+    }
+
+    var line_segment = segment.substring(segment_len, segment.length - 1)
+    segments.push(line_segment);
+
+    elem.innerHTML = original_text;
+
+    return segments;
+
+}
+
+function remove_element() {
+    var elem = document.getElementById('transcription')
+    for (var i = 0; i < 4; i++) {
+        document.getElementById("t" + i).remove();
+    }
+    elem.remove()
+}
+
```

Comment:
```suggestion
```

Picky style comment, remove the extra empty line, to be consistent with the rest of the codebase.